test/spec/generate_spec.py

```diff
@@ -63,7 +65,7 @@ def generate_topology(self):
         with open(self.topology_dot, 'r') as dot_file:
             dot_graph = dot_file.read()
             graph = graphviz.Source(dot_graph)
-            graph.render(self.topology_image, format='png', cleanup=True)
+            apa=graph.render(self.topology_image, format='png', cleanup=True)
```

Comment:
Since you changed my "Hej" log message, maybe we should keep it professional also in the variable names? ;-)

I don't see you using `apa` anywhere, so I assume the change is to hide a warning, in which case you could use `_ = graph.render(...)`.

Reply:
Good finding. It shouldn't be there, I used it for a quick test.